include/votca/tools/graphvisitor.h

```diff
@@ -61,7 +61,7 @@ class GraphVisitor {
   /// Edge(0,0) is a dummy value
  public:
   virtual void exploreNode(std::pair<Index, GraphNode>& vertex_and_node,
-                           Graph& graph, Edge edge = DUMMY_EDGE);
+                           Graph& graph, const Edge& edge = DUMMY_EDGE);
```

Comment:
what does the dummyedge do?

Reply:
instead of a default arguement, having two functions with different parameters is better, because it is easier to extend.